### PR TITLE
UI: grant submitted page

### LINKED ISSUE #254: UI: Grant creation approval notice after creation
After a grant is created, the user should be informed that their grant will not show up immediately and will only be displayed after review and approval.

**Requirements**
- After submitting a grant, user should be redirected to /dgrants/grant-submitted page (or other appropriate name)
- grant-submitted page should say "Thank you for your grant submission! \n Your grant has been queue'd for review and upon approval will be visible to everyone. \n Average time to review is less than 24 hours."
- No button should be necessary to go back home as the navigation should provide all that is necessary

### PR DESCRIPTION
**Pull request by mk1r**
resolves #254, new styling for grant submitted page will come later from @scco 

Screen Shot 2021-10-04 at 2 42 01 AM

### CODE REVIEW

**Comment by Ajand** [comment]
Screenshot from 2021-10-04 16-41-22
I also created this page for this issue

File: app/src/views/GrantSubmitted.vue [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<template>
+  <div>
+    <BaseHeader
+      name="Thank you for your grant submission! Your grant has been queue'd for review and upon approval will be visible to everyone. Average time to review is less than 24 hours."
```
Comment (mds1) [inline app/src/views/GrantSubmitted.vue:4]: typo 

```suggestion
      name="Thank you for your grant submission! Your grant has been queued for review and upon approval will be visible to everyone. Average time to review is less than 24 hours."
```

**Review by phutchins — APPROVED** [review]

**Review by scco — APPROVED** [review]
looks good to me.

**Comment by mds1** [comment]
@phutchins Medium term there probably needs to be some transparency/details on how the review process works:
1. If someone gets rejected, how do we contact them and tell them why?
2. How do they know if their grant was rejected vs. not yet reviewed?

And longer term need to figure out a more decentralized solution. Are we tracking this anywhere in a discussion or issue?

**Comment by mk1r** [comment]
@mds1 think it's just being tracked in #254 
@Ajand nice - what branch is that in?

**Comment by mk1r** [comment]
@mds1 how about I create a discussion for it then? and we close #254

**Comment by thelostone-mc** [comment]
@mk1r creating a discussion sounds good

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**